lib/mcb/commands/providers/withdraw_courses.rb

```diff
@@ -0,0 +1,11 @@
+summary "Withdraw courses for a pre determined list of providers"
+usage "withdraw_courses <provider code>"
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+  current_cycle = RecruitmentCycle.find_by(year: Settings.current_recruitment_cycle_year)
+  args.each do |argument|
```

Comment:
If you end up making other changes here anyway, it may be an idea to call the provider coder what it is:

```suggestion
   args.each do |provider_code|
```

Reply:
Done